packages/react-native-editor/README.md

```diff
@@ -133,34 +127,11 @@ and for Android:
 
 To run a single test instead of the entire suite, use `npm run native device-tests:local`. Here's an example that runs only `gutenberg-editor-paragraph.test`:
 
-`TEST_RN_PLATFORM=ios yarn device-tests:local gutenberg-editor-paragraph.test`
+`TEST_RN_PLATFORM=ios npm run native device-tests:local gutenberg-editor-paragraph.test`
 
 Note: You might experience problems that seem to be related to the tests starting the Appium server, e.g. errors that say `Connection Refused`, `Connection Reset` or `The requested environment is not available`. For now, you can manually start the Appium server via [appium desktop](https://github.com/appium/appium-desktop) or the CLI, then change the port number in the tests while (optionally) commenting out related code in the `beforeAll` and `afterAll` block. 
 
-For a more detailed outline of the UI tests and how to get started writing one, please visit the [UI Test documentation](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/__device-tests__/README.md) and our [contributing guide](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/__device-tests__/CONTRIBUTING.md).
-
-## Static analysis and code style
-
-The project includes a linter (`eslint`) to perform codestyle and static analysis of the code. The configuration used is the same as [the one in the Gutenberg project](https://github.com/WordPress/gutenberg/blob/master/eslint/config.js). To perform the check, run:
-
-```
-yarn lint
-```
-
-To have the linter also _fix_ the violations run: `yarn lint:fix`.
-
-In parallel to `eslint`, the project uses `Prettier` for codestyling. Run:
-
-```
-yarn prettier
-```
-to enforce the style. This will modify the source files to make them conform to the rules.
-
-`Flow` is used as a static type checker for JavaScript code. Flow checks JavaScript code for errors through static type annotations. These types allow you to tell Flow how you want your code to work, and Flow will make sure it does work that way. To perform the check run:
-
-```
-yarn flow
-```
+For a more detailed outline of the UI tests and how to get started writing one, please visit the [UI Test documentation](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/__device-tests__/README.md) and our [contributing guide](https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/__device-tests__/CONTRIBUTING.md).
```

Comment:
```suggestion
For a more detailed outline of the UI tests and how to get started writing one, please visit the [UI Test documentation](/packages/react-native-editor/__device-tests__/README.md) and our [contributing guide](/packages/react-native-editor/__device-tests__/CONTRIBUTING.md).
```

When using relative paths it will work with all branches.

Reply:
Thanks 👍